src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -114,7 +114,7 @@ export function registerTerminalActions() {
 		async run(accessor: ServicesAccessor) {
 			const terminalService = accessor.get(ITerminalService);
 			if (terminalService.isProcessSupportRegistered) {
-				const instance = terminalService.createTerminal();
+				const instance = terminalService.createTerminal({ target: terminalService.configHelper.config.defaultLocation });
```

Comment:
Let's make target optional and have it use defaultLocation if not specified  inside `createTerminal`